OracleWebCenterContent/dockerfiles/README.MD

```diff
@@ -0,0 +1,315 @@
+UCM_PORT=<port to be used for UCM managed server on container>
+IBR_PORT=<port to be used for IBR managed server on container>
+UCM_HOST_PORT=<host port to access UCM managed server - this is the port value to be used for -p option (left of the colon) sec. ### 5.1.3>
+IBR_HOST_PORT=<host port to access IBR managed server - this is the port value to be used for -p option (left of the colon) sec. ### 5.1.3>
+UCM_INTRADOC_PORT=<UCM intradoc port on container>
+IBR_INTRADOC_PORT=<IBR intradoc port on container>
+
+#component
+component=IPM,Capture,ADFUI
+
+#HOSTNAME
+HOSTNAME=<provide your host name>
+
+#Keep Container alive
+KEEP_CONTAINER_ALIVE=true
+```
+
+### 5.1.2. Admin Container (WCCAdminContainer)
+#### A. Creating and Running Admin Container
+
+Run the following command to create the Admin Server container:
+
+```
+docker run -it --name WCCAdminContainer --network=WCContentNET -p <Any Free Port>:<ADMIN_PORT> -v /scratch/wccdocker/volumes/wccontent:/u01/oracle/user_projects --env-file /scratch/wccdocker/webcenter.env.list oracle/wccontent:12.2.1.4.0 /bin/bash -c "/u01/oracle/container-scripts/createDomainandStartAdmin.sh; /bin/bash"
+
+# A sample command will look ike this -
+
+docker run -it --name WCCAdminContainer --network=WCContentNET -p 7001:7001 -v /scratch/wccdocker/volumes/wccontent:/u01/oracle/user_projects --env-file /scratch/wccdocker/webcenter.env.list oracle/wccontent:12.2.1.4.0 /bin/bash -c "/u01/oracle/container-scripts/createDomainandStartAdmin.sh; /bin/bash"
+```
+**Note:** The above command deletes any previous RCU with the same prefix if **DB_DROP_AND_CREATE=true**
+
+The docker run command creates the container as well as starts the Admin Server in sequence given below:
+
+* Node Manager
+* Admin Server
+
+When the command is run for the first time, we need to create the domain and configure the content managed servers, so following are done in sequence:
+
+* Loading WebCenter Content schemas into the database
+* Creating WebCenter Content domain
+* Extending WebCenter Content domain for associated products (e.g. IPM, ADFUI etc.) - based on **component** env varibale. 
+* Configuring Node Manager
+* Starting Node Manager
+* Starting Admin Server
+
+#### B. Stopping Admin Container
+```
+docker container stop WCCAdminContainer
+```
+
+#### C. Starting Admin Container
+```
+docker container start -i WCCAdminContainer
+```
+
+### 5.1.3. Managed Server Container (WCContentContainer)
+
+#### A. Creating and Running WCContent Container
+Run the following command to create the WCContent Managed Server container:
+
+```
+docker run -it --name WCContentContainer --network=WCContentNET -p <UCM_HOST_PORT>:<UCM_PORT> -p <IBR_HOST_PORT>:<IBR_PORT> -p <UCM_INTRADOC_PORT>:<UCM_INTRADOC_PORT> -p <IBR_INTRADOC_PORT>:<IBR_INTRADOC_PORT> --volumes-from WCCAdminContainer --env-file /scratch/wccdocker/webcenter.env.list oracle/wccontent:12.2.1.4.0  /bin/bash -c "/u01/oracle/container-scripts/configureOrStartWebCenterContent.sh; /bin/bash"
```

Comment:
I really dislike these `docker run` examples. Containers should run detached and should not have such wildly overloaded `CMD` directives.